client/src/components/new-request/main-container.vue

```diff
@@ -1,14 +1,18 @@
 <template>
-  <v-container class="main-container-style px-9 py-5 copy-normal">
+  <v-container fluid class="white rounded copy-normal">
     <v-row justify="space-between" class="submission-tabs-top">
       <slot name="container-header" />
-      <v-col cols="auto" class="pt-0" v-if="displayedComponent !== 'Success'">
+      <v-col cols="auto" class="py-0" v-if="displayedComponent !== 'Success'">
         <button id="back-to-search-btn"
```

Comment:
There are now 2 spans inside this button: the first if edit mode, else the second one.